packages/playout-gateway/src/connector.ts

```diff
@@ -81,4 +93,46 @@ export class Connector {
 			process.exit(0)
 		}, 10 * 1000)
 	}
+
+	/* eslint-disable @typescript-eslint/no-inferrable-types */
+	/* eslint-disable @typescript-eslint/no-empty-function */
+	private setupKoaEndpoints(): void {
+		const koaRouter: KoaRouter = new KoaRouter()
+		this.setupDevicesMakeReadyEndpoint(koaRouter)
+		this.setupDevicesStandDownEndpoint(koaRouter)
+
+		const port: number = 3009
```

Comment:
It would be nice being able to configure this with an env variable :)
E.g.
```typescript
const envPort: number = parseInt(process.env.REST_PORT)
const port: number = isNaN(envPort) ? 3009 : envPort

Reply:
Updated and extracted it to its own method